docs/getting-started/summaryofchanges.md

```diff
@@ -28,39 +28,50 @@ The following features and enhancements were added.
 #### API Mediation Layer
 - The API Catalog backend has been modified to support the OpenAPI 3.0 version. The API Catalog now supports the display of API documentation in the OpenAPI 3.0 format.
 
-- A new Eureka metadata definition has been developed to enable service registration that does not require using existing pre-prepared enablers. Both new and old metadata versions are supported by the Discovery Service. Corresponding documentation to onboard a service with the Zowe API ML without an onboarding enabler has also been refactored. 
+- A new Eureka metadata definition has been developed to enable service registration that does not require using existing pre-prepared enablers. Both new and old metadata versions are supported by the Discovery Service. Corresponding documentation to onboard a service with the Zowe API ML without an onboarding enabler has also been refactored.
 
 - The plain Java enabler has been redesigned for simple and straight-forward API service configuration. Configuration parameters have been refactored to remove duplicates and unused parameters, and improve consistency with other parameters. Documentation to Onboard a REST API service with the Plain Java Enabler (PJE) has also been refactored.
 
 
 #### Zowe App Server
-- 
-- 
+-
+-
 
 #### Zowe CLI
-- 
-- 
+
+- The Zowe CLI REST API now supports the following capabilities for managing data sets:
+  - Rename sequential and partitioned data sets. [#571](https://github.com/zowe/zowe-cli/issues/571)
+  - Migrate data sets. [#558](https://github.com/zowe/zowe-cli/issues/558)
+  - Copy data sets to another data set and copy members to another member. [#578](https://github.com/zowe/zowe-cli/issues/578)
+- The Zowe CLI REST API now supports HTTP ETags in response data. The ETag mechanism allows client applications to cache data more efficiently. ETAgs can also prevent simultaneous, conflicting updates to a resource. [#598](https://github.com/zowe/zowe-cli/issues/598)
+
+#### Zowe Explorer
```

Comment:
Here is a generic link to Explorer Changelog and a link to the Marketplace for download. Feel free to adjust if you prefer something else. 

The changelog to be update is here, if needed: 
https://github.com/zowe/vscode-extension-for-zowe/blob/master/CHANGELOG.md

Reply:
@IgorCATech I will go ahead and merge this into staging, feel free to make later changes if needed.